bazel/modules/rules_apk/README.md

```
@@ -0,0 +1,186 @@
+apk_go_deps.gazelle_override(
+    directives = ["gazelle:proto disable"],
+    path = "chainguard.dev/sdk",
+)
+```
+
+The referenced sdk code is unreachable at runtime — apko's auth.go is
+the only caller, and rules_apk's binaries never invoke auth.
+
+## Layout
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)